puppet/zulip/manifests/app_frontend_base.pp

```diff
@@ -68,7 +68,10 @@
   # The number of Tornado processes to run on the server;
   # historically, this has always been 1, but we now have experimental
   # support for Tornado sharding.
-  $tornado_processes = zulipconf('application_server', 'tornado_processes', 1)
+  # We need to convert the string returned by zulipconf to an integer. Puppet 5
+  # requires the awkward + 0 to achieve that. When we're fully on Puppet 6 on all
+  # supported platforms, this can be changed to the much nicer Integer() cast.
+  $tornado_processes = zulipconf('application_server', 'tornado_processes', 1) + 0
```

Comment:
`Integer()` works in Puppet 5, it’s just [documented](https://puppet.com/docs/puppet/5.5/function.html#conversion-to-integer) in a confusing place. I already used it in #14915.